models/api/coursework-material/questions.js

```diff
@@ -1,8 +1,8 @@
 const mongoose = require("mongoose");
-const {Schema} = mongoose;
-const {ObjectId} = Schema.Types;
+const { Schema } = mongoose;
+const { ObjectId } = Schema.Types;
 
-const options = {discriminatorKey: "kind"}
+const options = { discriminatorKey: "kind" }
```

Comment:
Since this is just a regular object literal, we'd need to decide if we want to do this spacing for _every other_ object literal we create in the project. Is that something we're ready to commit to? E.g.: `const person = { name: "Joe" }` vs. `const person = {name: "Joe"}`